languages/csharp/exercises/concept/chars/Chars.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+public static class Identifier
+{
+    public static string Clean(string str)
+    {
+        throw new NotImplementedException($"Please implement the Clean() method");
```

Comment:
We've slightly modified the format of the not implemented messages:

```suggestion
        throw new NotImplementedException($"Please implement the (static) Identifier.Clean() method");
```